pubsub-mapped-api/src/main/java/com/google/pubsub/clients/ProducerThread.java

```diff
@@ -0,0 +1,64 @@
+package com.google.pubsub.clients;
```

Comment:
Can we put this and The ProducerThreadPool class in a separate directory/package? Since they are more sanity-test code to ensure things are working, maybe something like com.google.pubsub.clients.tests.